integrations/pjrt/src/iree_pjrt/rocm/client.cc

```diff
@@ -17,16 +15,39 @@ ROCMClientInstance::ROCMClientInstance(std::unique_ptr<Platform> platform)
   // TODO: Get this when constructing the client so it is guaranteed to
   // match.
   cached_platform_name_ = "iree_rocm";
-  IREE_CHECK_OK(iree_hal_rocm_driver_module_register(driver_registry_));
 }
 
 ROCMClientInstance::~ROCMClientInstance() {}
 
 iree_status_t ROCMClientInstance::CreateDriver(iree_hal_driver_t** out_driver) {
-  iree_string_view_t driver_name = iree_make_cstring_view("rocm");
-  IREE_RETURN_IF_ERROR(iree_hal_driver_registry_try_create(
-      driver_registry_, driver_name, host_allocator_, out_driver));
-  logger().debug("ROCM driver created");
+  iree_string_view_t driver_name = iree_make_cstring_view("hip");
+
+  // Device params.
+  iree_hal_hip_device_params_t default_params;
+  iree_hal_hip_device_params_initialize(&default_params);
+
+  // Driver params.
+  iree_hal_hip_driver_options_t driver_options;
+  iree_hal_hip_driver_options_initialize(&driver_options);
+  driver_options.default_device_index = 0;
+
+  IREE_RETURN_IF_ERROR(
+      iree_hal_hip_driver_create(driver_name, &driver_options, &default_params, 
+                                 host_allocator_, out_driver));
```

Comment:
The code generally looks good to me, though the pre-commit lint check is requiring a few clang-format changes.